src/coreclr/gc/gcpriv.h

```diff
@@ -140,11 +140,11 @@ inline void FATAL_GC_ERROR()
 //
 // This means any empty regions can be freely used for any generation. For
 // Server GC we will balance regions between heaps.
-// For now disable regions for standalone GC and macOS builds
+// For now disable regions for standalone GC and iOS and tvOS builds
```

Comment:
```suggestion
// For now disable regions for iOS and tvOS builds
```
might as well take this opportunity to fix this comment since we do have a standalone GC with regions.